src/main/scala/uk/gov/nationalarchives/tdr/api/model/consignment/ConsignmentReference.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import uk.gov.nationalarchives.oci.Alphabet.Alphabet
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+object ConsignmentReference {
+  val alphabet: Alphabet = Alphabet.loadAlphabet(Right(IncludedAlphabet.GCRb25)) match {
+    case Left(error) => throw new Exception(s"Theres been an error creating the consignment reference", error)
+    case Right(alphabet) => alphabet
+  }
+
+  def createConsignmentReference(transferStartYear: Int, consignmentSequenceId: Long): String = {
+    val baseNumber = 25
```

Comment:
Could this value also be a constant, defined outside of the method? Just to make it easier to distinguish the parts that never change from the dynamic stuff.

Reply:
Ah yeah, didn't think of that. I'll change that :+1: